OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/menu/OpenTemplate.java

```diff
@@ -144,6 +144,14 @@ private Template getTemplateFromElement(NodeList templateNodes) {
 
     @Override
     public void actionPerformed(ActionEvent e) {
-        this.setVisible(true);
+        Enum canOpen = etc.canOpen();
```

Comment:
It could be a concrete Enum implementation - in this case "CanOpenState" variable